src/handlers/sdp/MediaSection.ts

```diff
@@ -691,6 +691,52 @@ export class OfferMediaSection extends MediaSection
 			? encoding.rtx.ssrc
 			: undefined;
 
+		for (const codec of offerRtpParameters!.codecs)
```

Comment:
```suggestion
		for (const codec of offerRtpParameters.codecs)
```

`!` not needed right?